addons/mail/static/src/models/attachment/attachment.js

```diff
@@ -184,13 +184,15 @@ function factory(dependencies) {
             if (!this.messaging) {
                 return;
             }
-            return this.messages.length
-                ? this.messages.some(message => (
+
+            if (this.messages.length && this.originThread && this.originThread.model == 'mail.channel') {
```

Comment:
`===`

Reply:
Also this is not correct here, we need an extra field `isDeletable` instead of re-using `isEditable` as we are now splitting the 2 behaviors.

Reply:
Actually looks like `isEditable` does not really make sense by itself on attachment, it's just badly named in the first place?